packages/storybook/src/stories/ino-button/ino-button.spec.ts

```diff
@@ -1,32 +1,41 @@
 import { expect, Locator, test } from '@playwright/test';
-import { goToStory } from '../test-utils';
+import { goToStory, setAttribute } from '../test-utils';
 
 test.describe('ino-button', () => {
   let inoButton: Locator;
 
+  async function getButtonDimensions() {
```

Comment:
I found out that playwright has a `boundingBox()` function which can be called on an element direktly.
https://playwright.dev/docs/api/class-elementhandle#element-handle-bounding-box

For instance: 
```
// height and width are numbers
const { height, width } = await inoButton.getByRole('button').boundingBox();
```